app/components/blacklight/gallery/slideshow_preview_component.rb

```diff
@@ -11,11 +11,23 @@ def initialize(document:, document_counter: nil, **args)
       end
 
       def before_render
-        thumbnail(presenter.thumbnail.render({ alt: presenter.heading })) if thumbnail.blank? && presenter.thumbnail.exists?
-        thumbnail(content_tag(:div, t('.missing_image', scope: [:blacklight_gallery]), class: 'thumbnail thumbnail-placeholder')) if thumbnail.blank?
+        set_default_thumbnail if thumbnail.blank?
         super
       end
 
+      def set_default_thumbnail
+        thumbnail_content = presenter.thumbnail.render({ alt: presenter.heading }) if presenter.thumbnail.exists?
+        unless thumbnail_content.present?
+          thumbnail_content = content_tag(
```

Comment:
should this be `tag.div`? https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/TagHelper.html#method-i-tag

Reply:
This was just a straight refactor. We change it to tag, but content_tag was there before and seems to work, so I'm not sure why we'd change it with this fix

Reply:
🤷 I don't care, this approach just seems to be obsolete if you follow the current rails docs.